.github/workflows/trigger-private-janus-build.yml

```diff
@@ -58,18 +58,20 @@ jobs:
       run: echo START_TIME=$(date "+%Y-%m-%dT%H:%M:%S") >> $GITHUB_ENV
 
     - name: Exchange GitHub App for GitHub Token
-      uses: wow-actions/use-app-token@d7957e08172ca2e8e49b35b8d266ad585885edc7 # pinned to the hash to guarantee immutability
-      id: generate_token
+      uses: actions/create-github-app-token@v1
+      id: app-token
       with:
         # These values are for the GitHub App guardian-janus-ci
-        # See https://github.com/organizations/guardian/settings/apps/guardian-janus-ci (only accesssible by GitHub owners)
-        app_id: ${{ secrets.GH_APP_ID }} # The ID of the GitHub App
-        private_key: ${{ secrets.GH_APP_PRIVATE_KEY }} # The private key of the GitHub App
-        
+        # See https://github.com/organizations/guardian/settings/apps/guardian-janus-ci (only accessible by GitHub owners)
+        app-id: ${{ secrets.GH_APP_ID }}
+        private-key: ${{ secrets.GH_APP_PRIVATE_KEY }}
+        owner: guardian
```

Comment:
Does it work to use an org as the owner? It isn't clear to me from the docs that that is the case. Most likely you've already tested this works!

Reply:
This was the only way I could [get this to work](https://github.com/guardian/janus-app/actions/runs/7575229176/job/20631402703). My understanding is the resulting token is scoped exactly to the `guardian/janus` repository. If one tried to use it against a different repository, a 403 would be returned.

<img width="719" alt="image" src="https://github.com/guardian/janus-app/assets/836140/74124945-9496-4923-adf4-e67a46d96a21">

Reply:
Yeah nice one, I can see the repositories key looks great. Thanks!